modules/gltf/editor/editor_scene_importer_gltf.cpp

```diff
@@ -60,6 +60,7 @@ Node *EditorSceneFormatImporterGLTF::import_scene(const String &p_path, uint32_t
 		}
 		return nullptr;
 	}
+	state->set_create_animations(bool(p_options["animation/import"]));
```

Comment:
Sometimes options can be empty.

Try doing a p_options.has("animation/import") check before accessing the dictionary.

Reply:
Right, I removed at some point because I saw p_options used elsewhere without checking.
But you're absolutely correct, better check anyway.
I just pushed a check